src/main/java/com/facebook/openwifirrm/RRMSchedule.java

```diff
@@ -21,7 +21,7 @@ public class RRMSchedule {
 	 * Scheduler (CronTrigger):
 	 * https://www.quartz-scheduler.org/documentation/quartz-2.4.0/tutorials/crontrigger.html
 	 */
-	public String[] cron;
+	public List<String> cron;
```

Comment:
Oh, forgot to write this last time - if you rename this to `crons` it will avoid any compatibility issues/crashing (and more appropriate name anyway).